### ISSUE TITLE
Column, Line down/up shortcut

### ISSUE THREAD

**Issue by Khaztaroth** [SubtitleEdit/subtitleedit#9080]
I've using the program a lot with automated transcriptions and sometimes lines need to be moved up/down in the column order.

I might be using the shortcut wrong but these result in the text overriding the next/previous line
![image](https://github.com/user-attachments/assets/55f07d17-1c5d-4cfc-83ef-8490706aa70a)
My current workaround is to copy and then immediately paste the line as a way to shift its position without altering the timing to then be modified. This becomes relevant if I'm trying to extend a line to the next line, since it will take the next line in the column and not the line that is timed after it. 

Is there a way to shift the line itself and not just the text? If not how possible is it to be added?

**Comment by niksedk** [comment]
So you would like a `Move current line up` / `Move current line down`  ?

**Comment by Khaztaroth** [comment]
Yes, if possible, just shift them up/down in the list without other changes.
